refinery/core/api.py

```diff
@@ -498,6 +498,13 @@ class Meta:
         }
 
     def dehydrate(self, bundle):
+        version_details = bundle.obj.get_version_details()
+
+        if version_details is None:
```

Comment:
[Tastypie docs](http://django-tastypie.readthedocs.io/en/latest/resources.html#flow-through-the-request-response-cycle) say that serialization happens after objects are retrieved, so it appears to be impossible to have an incomplete data set at this point. Even if that's somehow not the case, returning an incomplete dataset is not the answer.